logging/testing.go

```diff
@@ -0,0 +1,55 @@
+package logging
+
+import (
+	"strings"
+	"testing"
+
+	"go.uber.org/zap/zapcore"
+)
+
+type testAppender struct {
+	tb testing.TB
+}
+
+// NewTestAppender returns a logger appender that logs to the underlying `testing.TB` object.
+func NewTestAppender(tb testing.TB) Appender {
+	return &testAppender{tb}
+}
+
+// Write outputs the log entryt o the underlying test object `Log` method.
+func (tapp *testAppender) Write(entry zapcore.Entry, fields []zapcore.Field) error {
+	tapp.tb.Helper()
+	const maxLength = 10
+	toPrint := make([]string, 0, maxLength)
+	// toPrint = append(toPrint, entry.Time.Format(appender.timeFormatStr))
```

Comment:
Yeah I can unify these time format strings once module logging goes back in review.

Reply:
Oops, I never got back to making this "real".

I'll take a stab here. The only intention of this was so the `ConsoleAppender` could be testing aware. But with this patch we don't use the `ConsoleAppender` for testing. So I'll look into removing the variable altogether for a single constant.

I'd like* to remove `inUTC` as well now (the intention being tests use local time and everything else is UTC). So I'll look into that now also.